wic/src/main/webapp/WEB-INF/views/admin/conceptText/addConceptText.jsp

```diff
@@ -0,0 +1,158 @@
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c"%>
+<%@ taglib prefix="fn" uri="http://java.sun.com/jsp/jstl/functions" %>
+<%@ taglib prefix="spring" uri="http://www.springframework.org/tags"%>
+<%@ taglib prefix="form" uri="http://www.springframework.org/tags/form"%>
+<script src="https://unpkg.com/easymde/dist/easymde.min.js"></script>
+<link type="text/css" rel="stylesheet" href="https://unpkg.com/easymde/dist/easymde.min.css">
+<link type="text/css" href="<c:url value="/resources/DigInGIconPack/diging-icon-pack.css" />" rel="stylesheet">
+
+
+<form:form method='POST' action="/wic/admin/text/add" modelAttribute="conceptTextFormData" onsubmit="return verifyNonEmpty()">
```

Comment:
`wic` cannot be hard-coded. There is no guarantee the app will always be deployed as `wic`.

Reply:
(e.g. in the staging instance it's not)